cms/static/js/spec/video/transcripts/file_uploader_spec.js

```diff
@@ -5,7 +5,7 @@ define(
         "xmodule", "jquery.form", "jasmine-jquery"
     ],
 function ($, _, Utils, FileUploader) {
-    describe('Transcripts.FileUploader', function () {
+    xdescribe('Transcripts.FileUploader', function () {
```

Comment:
I'd probably add a TODO so we remember to re-enable this. I don't think I'd notice the 'x'.